glyphs2fontir/src/source.rs

```diff
@@ -330,11 +196,15 @@ impl Work for GlyphIrWork {
         // Glyphs have layers that match up with masters, and masters have locations
         let mut axis_positions: HashMap<String, HashSet<NormalizedCoord>> = HashMap::new();
         for layer in glyph.layers.iter() {
-            let Some(master_idx) = self.master_indices.get(&layer.layer_id) else {
+            let Some(master_idx) = font_info.master_indices.get(layer.layer_id.as_str()) else {
+                // If the glyph has an associatedMasterId this is fine, if not ... not
```

Comment:
i don't understand the wording of this inline comment.. "this is fine, if not ... not"
Actually it should be the layer that may/may not have associatedMasterId, not the glyph per se.
Also you may want to mark this with a TODO and link to the associated issue #62

Reply:
I see that in the following commit you're correctly checking for associated_master_id in the layer, not the glyph, but the comment looks stale

Reply:
fixed comment